main.go

```diff
@@ -119,6 +120,22 @@ func sendAlert(event *corev2.Event) error {
 		environment = event.Entity.Namespace
 	}
 
+	var attributes = make(map[string]string)
+	for key, value := range event.Entity.Labels {
+		if !strings.ContainsAny(key, ".$") {
```

Comment:
Just replace "." with an underscore ("_"). That's what happens for other integrations.

Reply:
Good suggestion. I've updated the code to replace the disallowed chars rather than ignore the labels.